tests/integration/bond_test.py

```diff
@@ -366,6 +366,15 @@ def test_create_linux_bridge_over_bond(bond99_with_slave):
         assertlib.assert_state(desired_state)
 
 
+@pytest.mark.xfail(strict=True, reason="Jira issue # NMSTATE-272")
```

Comment:
Sorry, there is a small nit I missed before. Please use a full URL to the issue to make it easier to open. Thank you!

Reply:
Done